src/Models/EntityWrapper.cs

```diff
@@ -56,7 +57,18 @@ public Vector3 Pos
                 return new Vector3(p.X, p.Y, p.Z);
             }
         }
-    
+
+        public int Distance
+        {
+            get
+            {
+                var p = GetComponent<Render>();
+                var player = GameController.Instance.Player;
+                var distance = Math.Sqrt(Math.Pow(player.Pos.X - p.X, 2) + Math.Pow(player.Pos.Y - p.Y, 2));
```

Comment:
would be nice if we can include the Z axis in this calculation..... basically Z axis projection in the 2D plane.